addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -229,8 +234,15 @@ class DescriptionScreen extends Component {
      * @param {Object} ui an object with label and value properties for
      *      the selected option.
      */
-    _selectIndustry(ev, ui) {
+    async _selectIndustry(ev, ui) {
```

Comment:
Is the `async` really considered in the caller (jquery autocomplete select property)?
Rather than ensuring it is, do we actually need it?
As this is about sending analytics data which we don't even do anything with the return call, we can probably not await it at all?